server/graphql/v1/inputTypes.js

```diff
@@ -274,6 +286,7 @@ export const TierInputType = new GraphQLInputObjectType({
       description: 'amount that you are trying to raise with this tier',
     },
     password: { type: GraphQLString },
+    settings: { type: GraphQLJSON },
```

Comment:
If we support it in the Input, we should ask for `customFields`, not settings. We don't want to have to take care of implementing a merging strategy when updating `settings`.

Reply:
Not sure I understand this completely, following the API spec:

> Add a new settings (JSON) column to Tiers that will contain additional data & Expose settings through GraphQL

What is changing now?

Reply:
You can discard what I wrote, the way it is implemented it will add complexity for not much gain.

Just let's make sure we don't lose the settings when we update the `Tier` from the interface. (it could be easy to introduce a bug that is resetting the settings in that case, because we don't surface settings in in the inputs).